buildSrc/src/main/java/org/elasticsearch/gradle/info/GlobalBuildInfoPlugin.java

```diff
@@ -276,6 +278,28 @@ private static String findJavaHome(String version) {
         return versionedJavaHome;
     }
 
+    private static boolean findIfBundledJdkSupported(Project project) {
+        if (Os.isFamily(Os.FAMILY_UNIX) == false || Os.isFamily(Os.FAMILY_MAC)) {
```

Comment:
We should cache this result in a static variable so we don't have to shell out on every build invocation. This is considered safe because a) we're just storing a static reference to a `boolean` so leaks aren't an issue and b) this is _very_ unlikely to change between build invocations on the same machine.

We do the same for `findDefaultParallel()` so let's follow that pattern.